publications-2018.bib

```diff
@@ -990,3 +990,16 @@ @PhdThesis{SafinThesis
 }
 
 @Comment{jabref-meta: databaseType:bibtex;}
+
+
+@article{
+  author = {Ira Neitzel, Thomas Wick, Winnifried Wollner},
```

Comment:
That's unfortunately not the format in which bibtex wants information. That programs wants you to say this:
```suggestion
  author = {Ira Neitzel and Thomas Wick and Winnifried Wollner},
```